Library/Homebrew/utils/pypi.rb

```diff
@@ -54,21 +47,38 @@ def get_pypi_info(package, version)
     sdist = json["urls"].find { |url| url["packagetype"] == "sdist" }
     return json["info"]["name"] if sdist.nil?
 
-    [json["info"]["name"], sdist["url"], sdist["digests"]["sha256"]]
+    [json["info"]["name"], sdist["url"], sdist["digests"]["sha256"], json["info"]["version"]]
   end
 
   # Return true if resources were checked (even if no change).
-  def update_python_resources!(formula, version = nil, print_only: false, silent: false,
-                               ignore_non_pypi_packages: false)
-
-    if !print_only && AUTOMATIC_RESOURCE_UPDATE_BLOCKLIST.include?(formula.full_name)
-      odie "The resources for \"#{formula.name}\" need special attention. Please update them manually."
-      return
+  def update_python_resources!(formula, version: nil, package_name: nil, extra_packages: nil, exclude_packages: nil,
```

Comment:
Since this changed from `version = nil` to `version: nil`, `bump-formula-pr.rb` line 339 needs to be changed to
```
resources_checked = PyPI.update_python_resources! formula, version: new_formula_version,
```
to support for instance, `brew bump-formula-pr --force -w ipython --version 7.19.0`

Reply:
Ah, good catch, thanks!

(Changed in the `update-python-resources` command but not in `bump-formula-pr`)